website/index.html

```diff
@@ -132,68 +133,90 @@
-                <p class="text-left">Penn State University</p>
-                <p class="text-left">Business Building</p>
+            <div class="logo-holder">
+                <img src="assets/images/logo-all-blue-no-padding.svg" id="hackpsu-logo"
+                class="animated rubberBand"
+                alt="hackpsulogo">
+            </div>  
+        </div>
+    
+        <!-- Event info and mountain graphics -->
+        <div class="row header-content">
+          <!-- Header info (event date, location) and CTA -->
+          <div class="col-md-5">
+            <div class="row event-info">
+              <div class="offset-md-2 col-md-8 ">
+                <div class="row">
+                  <div class="col-12 header-text wrapped-box">
+                    <p class="text-left"><span>October 6-7</span></p>
+                    <p class="text-left"><span>Penn State University</span></p>
+                    <p class="text-left"><span>Business Building</span></p>
+                  </div>
+                </div>
               </div>
             </div>
-            <div class="row bottom">
-              <a href="https://app.hackpsu.org">
-                <div class="col-md-12 header-text align-bottom header-hover">
-                  <p class="text-left">Register Now</p>
-                  <div class="bottom-bar"></div>
+
+            <div class="row cta">
+                <div class="offset-md-2 col-lg-8 col-md-10 col-sm-12 register-btn">
+                  <a href="https://app.hackpsu.org">
+                    <div class="header-text text-center">
+                      <p class="text-center">Register Now</p>
+                    </div>
+                  </a>
                 </div>
-              </a>
+            </div>
+          </div>
+
+          <!-- Mountain vista graphics -->
+          <div class="col-md-7">
+            <div class="row header-graphic">
+              <img src="assets/images/flat-mountain-graphic-snow.svg" class="img-fluid mountain">
+              <!-- Each cloud img needs its own div to keep their position correct on resize 
+              (abosolute/relative positioning weirdness). -->
+              <div class="cloud-1">
+                  <img src="assets/images/cloud-2.svg" class="floating">
+              </div>
+              <div class="cloud-2">
+                  <img src="assets/images/cloud-1.svg" class="floating">
+              </div>
+              <div class="cloud-3">
+                <img src="assets/images/cloud-5.svg" class="floating">
+              </div>
+
             </div>
           </div>
 
         </div>
       </div>
 
-      <div class="section-separator-first">
-        <img class="section-separator-background" src="assets/images/border1_wTransparency.svg">
-      </div>
     </header>
+
     <section id="about">
       <div class="container">
+
         <div class="row">
-          <div class="section-header col-xs-12">
+          <div class="section-header col-md-12">
```

Comment:
What happens on smaller devices? Shouldn't this span the width for all?

Reply:
Fixed in latest commits - just used generic col-12